models/users.js

```diff
@@ -27,6 +31,46 @@ const { formatUsername } = require("../utils/username");
 const { logType } = require("../constants/logs");
 const { addLog } = require("../services/logService");
 
+const archiveUsers = async (usersData) => {
```

Comment:
Write doc for this function. Take reference from addOrUpdate.

Reply:
Done